src/pfs/pfs.proto

```diff
@@ -305,6 +305,20 @@ message ClearCommitRequest {
   Commit commit = 1;
 }
 
+message SquashCommitRequest {
+  Commit commit = 1;
+  bool recursive = 2;
```

Comment:
the commit field is self explanatory, but could you add a comment here (and for the DropCommitRequest message) for the recursive field.